packages/react/src/ReactElementValidator.js

```diff
@@ -307,12 +307,6 @@ export function createElementWithValidation(type, props, children) {
 
   const element = createElement.apply(this, arguments);
 
-  // The result can be nullish if a mock or a custom function is used.
-  // TODO: Drop this when these are no longer allowed as the type argument.
```

Comment:
I'm curious about this line. createElement might always return a ReactElement in React code, but what code is mocking this in a way that would return null?

Reply:
@nhunzaker From this [commit](https://github.com/facebook/react/commit/c6b2687bbda0c9d828be51d3d4908adb89d52a09#diff-ff241f17a116412c6d63b7bfab8d68d6R227), look like we support custom element validator?

Reply:
@sebmarkbage Could you give some info about this?

Reply:
@sebmarkbage Could you give some info about this?

Reply:
@sebmarkbage Could you give some info about this?